### ISSUE TITLE
Java cant execute the class file

### ISSUE THREAD

**Issue by JavierReyes945** [JavierReyes945/lgbc#1]
Following the command to build and execute from the @drujensen repository, but error at execution is thrown.

## Branch

[java_fib](https://github.com/JavierReyes945/lgbc/tree/java_fib)

## Log
```
$ javac java/fibonacci/Fibonacci.java 

$ java java.fibonacci.Fibonacci 
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.SecurityException: Prohibited package name: java.fibonacci
        at java.base/java.lang.ClassLoader.preDefineClass(ClassLoader.java:898)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1014)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:802)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:700)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:623)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at java.base/sun.launcher.LauncherHelper.loadMainClass(LauncherHelper.java:760)
        at java.base/sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:655)

```

**Comment by drujensen** [comment]
@JavierReyes945 I think the package `java` is reserved.  Try moving it to a different directory/package name and see if that fixes your issue.

**Comment by JavierReyes945** [comment]
> @JavierReyes945 I think the package `java` is reserved. Try moving it to a different directory/package name and see if that fixes your issue.

Wow, sorry for the tag, It was supposed to be merely informative, but not that you needed to check this. Thanks a lot for the comment (will check that now), and specially thanks for your fib repository. It was quite cool.

**Comment by JavierReyes945** [comment]
Fixed with b5f7b9a062fa763b40fc1ec243162905817b39a9
